### PR TITLE
[Search Experiences] Improve Eland hugging face notebook

### PR DESCRIPTION
**Pull request by saarikabhasi**
Adding some improvements in eland hugging face notebook. Changes added: 

- [x] Remove outputs
- [x] Created a simple blog.json with titles document and Removed fetching blogs.zip file
- [x] Update description and heading
- [x] Update mappings and indexing

### CODE REVIEW

File: notebooks/integrations/hugging-face/loading-model-from-hugging-face.ipynb [hunk 1]
```diff
@@ -490,209 +261,31 @@
...
-    }
-   ],
-   "source": [
-    "BLOGS_FILE=\"data/2023-07-27-blogs-no-embeddings.json\"\n",
-    "df_blogs_embeddings = pd.read_json(BLOGS_FILE, lines=True)\n",
-    "\n",
-    "def gen_rows(df):\n",
-    "  for doc in df.to_dict(orient='records'):\n",
-    "    yield doc\n",
+    "url = \"https://raw.githubusercontent.com/elastic/elasticsearch-labs/main/notebooks/hugging-face/blogs.json\"\n",
```
Comment (saarikabhasi) [inline notebooks/integrations/hugging-face/loading-model-from-hugging-face.ipynb:280]: Tested locally and works. 
Colab will work when this Pr is merged (`blogs.json` will exist in  main `elasticsearch-labs` repo).

**Comment by joemcelroy** [comment]
this is great thanks saarika!

one minor change: can we provide the output for "querying the dataset" please?

**Comment by saarikabhasi** [comment]
Question @joemcelroy  Would it be safe to delete [2023-07-27-blogs-no-embeddings.json.zip](https://github.com/elastic/elasticsearch-labs/blob/main/notebooks/integrations/hugging-face/2023-07-27-blogs-no-embeddings.json.zip) file ?

**Comment by joemcelroy** [comment]
> Question @joemcelroy Would it be safe to delete [2023-07-27-blogs-no-embeddings.json.zip](https://github.com/elastic/elasticsearch-labs/blob/main/notebooks/integrations/hugging-face/2023-07-27-blogs-no-embeddings.json.zip) file ?

yes delete please! thanks

**Review by joemcelroy — APPROVED** [review]

**Pull request merged**